app/views/casa_cases/show.html.erb

```diff
@@ -50,7 +50,7 @@
 
     <div>
       <h6><strong>Assigned Volunteers:</strong></h6>
-      <% policy_scope(@casa_case.volunteers).each_with_index do |volunteer, index| %>
+      <% assigned_volunteers(policy_scope(@casa_case)).each_with_index do |volunteer, index| %>
```

Comment:
policy has to be on the outside whoops

Reply:
has this been resolved @compwron and was it included in our deploy to prod last week? If yes, I'll move it over to Done so we know it's complete. 😃

Reply:
@Zrrrpy yep this is in prod